spartan/terraform/deploy-metrics/main.tf

```diff
@@ -23,7 +23,7 @@ provider "google" {
 
 resource "google_compute_address" "grafana_ip" {
   provider     = google
-  name         = "grafana-ip-${var.RELEASE_NAME}"
+  name         = "grafana-ip"
```

Comment:
I had to rename these resources back to their original names otherwise tf would attempt to destroy and recreate losing the IP addresses. Grafana would be fine but losing the collector's IP address would mean having to redeploy everything from TF in order to use the new address.

Unfortunately GCP addresses can not be renamed either so we're stuck with them until we migrate everything to use proper domain name (TBD)